src/hooks/tests/useToggle.test.tsx

```diff
@@ -19,7 +20,7 @@ const resetHandlerMocks = () => {
 };
 
 // eslint-disable-next-line react/prop-types
```

Comment:
nit: is this eslint disable comment still relevant?

Reply:
Good question! It looks like it's still there on latest `release-22.x` as of writing https://github.com/openedx/paragon/blob/c5c48aa2aed9c4e583d83bbe58783a93db40977a/src/hooks/tests/useToggle.test.tsx#L22

I made an issue to go through and clean up any of these we don't need https://github.com/openedx/paragon/issues/3391